server/middlewares/validations/articleValidator.js

```diff
@@ -0,0 +1,34 @@
+import models from '../../models';
+import {
+  ARTICLE_NOT_FOUND
+} from '../../helpers/responseMessages';
+
+const { Article } = models;
+
+/**
+ * @description Articles ValidatorClass
+ * @class
+ */
+class ArticleValidator {
+  /**
+   * @description Check if an article exists
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   * @returns {function} next
+   */
+  static async articleExistInDatabase(req, res, next) {
+    try {
+      const { articleId } = req.params;
+      const dbResult = await Article.findOne(articleId);
+      if(dbResult) return next();
+      return res.status(404).json({
```

Comment:
Whats the reason behind our lack of the use of the "else" statement?

Reply:
Okay. If `if..else` was used here, ESLint complains and says the function needs a return statement

Reply:
Where is the reason?